models/feature_flag.py

```diff
@@ -0,0 +1,15 @@
+from main import db, cache
+
+class FeatureFlag(db.Model):
+    __tablename__ = 'feature_flags'
+    name = db.Column(db.String, primary_key=True)
+    enabled = db.Column(db.Boolean, nullable=False)
+
+    def __init__(self, name, enabled=False):
+        self.name = name
+        self.enabled = enabled
+
+    @classmethod
+    @cache.cached(timeout=30, key_prefix='feature_flags_get')
```

Comment:
I think you need to use [cache.memoize](https://pythonhosted.org/Flask-Cache/#flask.ext.cache.Cache.memoize) here to take the function arguments into account.

This also means you can do [cache.delete_memoized](https://pythonhosted.org/Flask-Cache/#flask.ext.cache.Cache.delete_memoized) in the admin interface to invalidate immediately.